### DISCUSSION TITLE
There is an errror in your sql

### DISCUSSION THREAD

**Discussion by EmanYm** [ericgrandt/TotalEconomyPaper#12] [Q&A]
I was wondering why half my plugins were failing to load, [then I found this in my console](https://www.toptal.com/developers/hastebin)

To fix it, I am trying to import the schema.sql into my database and I get this error

> MySQL said: Documentation
>
>#1064 - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version >for the right syntax to use near '(uuid()),
>    currency_id INT NOT NULL UNIQUE,
>    default_balance DECIMAL(38,' at line 2

Edit:
I suspect the version of MySQL available to me on my server host is different than the version you are building your schema against.
Clicking on the documentation on my database panel takes me to MySQL 5.7 reference manual.

**Comment by ericgrandt** [comment]
Hi @EmanYm, you're correct in your edit. Unfortunately, TotalEconomy requires MySQL 8 (in particular, >= 8.0.13 I believe) at the minimum.
